openmetadata-ui/src/main/resources/ui/src/components/Settings/Users/UsersProfile/UserProfileDetails/UserProfileDetails.component.tsx

```diff
@@ -306,13 +300,32 @@ const UserProfileDetails = ({
       </Space>
     ),
     [
-      userData.personas,
-      hasPersonaEditPermission,
       defaultPersona,
+      userData.personas,
+      hasEditPermission,
       handleDefaultPersonaUpdate,
     ]
   );
 
+  const handleRestoreUser = useCallback(async () => {
+    setIsLoading(true);
+
+    try {
+      await restoreUser(userData.id);
+      afterDeleteAction(true);
```

Comment:
Here we are restoring the soft-deleted user. Then, we are setting the soft-deleted flag to `true` in `afterDeleteAction(true)`. I think it should be set to `false`, right?

Reply:
No, it will changes the `prev` state which is `true to false`.  Have also added the comment on the restore call for the same.

<img width="575" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/66266464/744fdd34-e5f3-45e8-89b1-9791ccf010a1">